src/_P042_Candle.ino

```diff
@@ -76,422 +83,440 @@ enum ColorType {
+                # ifndef BUILD_NO_DEBUG
 
-          if (!val_Bright.isEmpty()) {
-             if (val_Bright.toInt() > -1 && val_Bright.toInt() < 256) {
-                PCONFIG(3) = val_Bright.toInt();     // Brightness
-                Candle_bright = PCONFIG(3);
-                #ifndef BUILD_NO_DEBUG
-                if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
-                  String log = F("CAND : CMD - Bright : ");
-                  log += val_Bright;
-                  addLogMove(LOG_LEVEL_DEBUG, log);
-                }
-                #endif
-             }
+            if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
+              addLogMove(LOG_LEVEL_DEBUG, concat(F("CAND : CMD - Type : "), val_Type));
+            }
+                # endif // ifndef BUILD_NO_DEBUG
           }
+        }
+
+        if (!val_Bright.isEmpty()) {
+          if ((val_Bright.toInt() > -1) && (val_Bright.toInt() < 256)) {
+            PCONFIG(3)    = val_Bright.toInt(); // Brightness
+            Candle_bright = PCONFIG(3);
+                # ifndef BUILD_NO_DEBUG
 
-          if (!val_Color.isEmpty()) {
-            long number = strtol( &val_Color[0], nullptr, 16);
-            // Split RGB to r, g, b values
-            uint8_t r = number >> 16;
-            uint8_t g = number >> 8 & 0xFF;
-            uint8_t b = number & 0xFF;
-
-            PCONFIG(0) = r;   // R
-            PCONFIG(1) = g;   // G
-            PCONFIG(2) = b;   // B
-            Candle_red = PCONFIG(0);
-            Candle_green = PCONFIG(1);
-            Candle_blue = PCONFIG(2);
-            PCONFIG(5) = 1;
-            Candle_color = (ColorType)PCONFIG(5);   // ColorType (ColorSelected)
-
-            #ifndef BUILD_NO_DEBUG
             if (loglevelActiveFor(LOG_LEVEL_DEBUG)) {
-              String log = F("CAND : CMD - R ");
-              log += r;
-              log += F(" G ");
-              log += g;
-              log += F(" B ");
-              log += b;
-              addLogMove(LOG_LEVEL_DEBUG, log);
+              addLogMove(LOG_LEVEL_DEBUG, concat(F("CAND : CMD - Bright : "), val_Bright));
             }
-            #endif
-          } else {
-            PCONFIG(5) = 0;
-            Candle_color = (ColorType)PCONFIG(5);   // ColorType (ColorDefault)
-            #ifndef BUILD_NO_DEBUG
-            addLog(LOG_LEVEL_DEBUG, F("CAND : CMD - Color : DEFAULT"));
-            #endif
+                # endif // ifndef BUILD_NO_DEBUG
           }
+        }
+
+        if (!val_Color.isEmpty()) {
+          long number = strtol(&val_Color[0], nullptr, 16);
+
+          // Split RGB to r, g, b values
+          uint8_t r = number >> 16;
+          uint8_t g = number >> 8 & 0xFF;
+          uint8_t b = number & 0xFF;
+
+          PCONFIG(0)   = r; // R
+          PCONFIG(1)   = g; // G
+          PCONFIG(2)   = b; // B
+          Candle_red   = PCONFIG(0);
+          Candle_green = PCONFIG(1);
+          Candle_blue  = PCONFIG(2);
```

Comment:
Why store it first in an uint8_t, then in a PCONFIG and then copy it to `Candle_xxx` ?
Why not directly into either `PCONFIG` or this `Candle_xxx` value? (I know you also need to do `& 0xFF` )